msal4j-sdk/src/samples/msi-sample/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.microsoft.identity</groupId>
+    <artifactId>msi-function-java</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Azure Java Functions</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
+        <azure.functions.maven.plugin.version>1.24.0</azure.functions.maven.plugin.version>
+        <azure.functions.java.library.version>3.0.0</azure.functions.java.library.version>
+        <functionAppName>msi-function-java-1687284411707</functionAppName>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>msal4j</artifactId>
+            <version>1.13.8</version>
```

Comment:
Does the MSI ~sample~  dev app use MSAL4J 1.13.8 fixed version?

Reply:
No, the latest jar that is created.

Reply:
Through IDE magic IntelliJ and Maven will either grab the version of the code you have in the editor or the version you built locally rather than trying to get the 'real' 1.13.8 version from the public package repo. We usually update these version numbers as part of a release, and for development they just need to match the version number in your local copy of msal4j's pom.xml